add back to start button to confirmation page

axemonkey opened the pull request:
[Pivotal](https://www.pivotaltracker.com/story/show/116519385)

A back to start button, on the confirmation page.

![screen shot 2016-03-29 at 15 57 58](https://cloud.githubusercontent.com/assets/988436/14112558/07395e80-f5c7-11e5-9538-617b9f8fcbd3.png)

NB: links to `root_path`, which works fine locally and probably on demo/staging envs, but will probably need to do something different when it goes properly live. I think?

novotnyjakub commented:
Sorry @axemonkey I've merged the refund confirmation page PR and this will probably need a rebase. Hopefully shouldn't be too painful.

novotnyjakub commented:
:+1: otherwise it looks good!

axemonkey commented:
Rebased and sorted, no worries. :+1: 

@novotnyjakub I added the `question` tag to this as I couldn't remember what you'd said about `root_path` for other envs, and did we need to do any more for that in this PR?

novotnyjakub commented:
I think this is all for this particular story. We will need to do a bit of work when we get the "static GDS homepage", which will require some small changes in multiple places here.

axemonkey commented:
Cool. In that case, I'll merge.
